components/header.tsx

```diff
@@ -58,8 +58,8 @@ const Header = () => {
     });
 
     const items = [
-        { id: TabIds.HOME, title: 'Home' },
-        { id: TabIds.PRODUCTS, title: 'Products' },
+        { ariaControls: 'home', id: TabIds.HOME, title: 'Home' },
```

Comment:
curiously, BigDesign lists this field as optional but is throwing a warning from https://github.com/bigcommerce/big-design/blob/master/packages/big-design/src/components/Tabs/Tabs.tsx#L40. I'm adding this in for now, but doesn't seem to fit our particular use case of using tabs for navigation.